symplyphysics/laws/kinematic/vector/velocity_of_transfer_between_reference_frames.py

```diff
@@ -0,0 +1,67 @@
+from symplyphysics import (
+    units,
+    angle_type,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    add_cartesian_vectors,
+    scale_vector,
+    cross_cartesian_vectors,
+)
+
+# Description
+## Suppose two reference frames, one of which is fixed (S) and the other one is moving (S'). The movement of
+## a body stationary in moving frame S' due to the movement of the frame itself is called transfer movement.
+## The velocity related to such movement is called transfer velocity. For any material point X, its transfer
+## velocity relative to fixed frame S with origin O is the sum of the velocity of moving origin O' of frame S'
+## (since S' is moving) and the cross product of the angular velocity of moving frame's rotation and the position
+## vector of X in moving frame S'.
+
+# Law: v_tr = v_0 + cross(w, r)
+## v_tr - vector of transfer velocity of point X relative to fixed frame S.
+## v_0 - vector of velocity of moving origin O' of frame S' relative to fixed frame S
+## w - pseudovector of angular velocity related to rotation of moving frame S' about instantaneous axis
+## r - vector of position of point X relative to moving frame S'
+
+
+def transfer_velocity_law(
+    origin_velocity_: Vector,
```

Comment:
Since origin (and axes) define reference frame, why don't we use `frame_velocity` here, instead of `origin_velocity`. Or `relative_frame_velocity` to be clear.
This way we avoid confusion in understanding, what is `origin velocity` and what is `origin` in general.

Reply:
To me it's more confusing what _frame velocity_ is. And it makes little sense from the derivation, where the two origins (the static one and the moving one) are used to obtain this formula.

Reply:
I'll give it a try with `frame` though.